modules/nf-core/sentieon/dnamodelapply/main.nf

```diff
@@ -0,0 +1,81 @@
+process SENTIEON_DNAMODELAPPLY {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    container 'nf-core/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(vcf), path(idx)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    path ml_model
```

Comment:
Should there also be a `meta`-field for the `ml_model`?  (I guess so, but I don't know for sure.)

Reply:
No harm in adding one I suppose 😄

Reply:
It seems that you committed the change on the wrong branch 🤔

https://github.com/nf-core/modules/pull/3733

Not that it matters much. I guess can just approve both PRs.

Reply:
That's for dnascope, which also uses ml_model file :)

Reply:
Ah 😆  my bad

Reply:
Seems like I can only handle one Sentieon PR at a time 😆

Reply:
No worries 😄

Reply:
On the other hand, I have been bombarding you with PR reviews 😆

Reply:
You are on a good roll 😁 👏